src/modules/mod.rs

```diff
@@ -130,6 +133,32 @@ pub fn handle<'a>(module: &str, context: &'a Context) -> Option<Module<'a>> {
         }
     };
 
+    let (time_limit, user_specified) = match context.module_timeout(module) {
+        Some(dur) => (dur, true),
+        None => (Duration::from_millis(3000), false),
```

Comment:
I'd prefer a default timeout closer to the current command timeout of 500ms.

Reply:
I've set it to 700, lmk if that sounds ok.